lib/src/chart/line_chart/line_chart_data.dart

```diff
@@ -198,4 +204,62 @@ class FlDotData {
     this.dotSize = 4.0,
     this.checkToShowDot = showAllDots,
   });
-}
\ No newline at end of file
+}
+
+class DashDefinition {
```

Comment:
Please remove it, we can add line style functionality line Dash and Dot effects in next phase, here we just add two feature (Extra Lines -> you use it for avg line, and BelowBar Lines-> you use it for below bar lines), then I will add LineStyle after this PR, don't worry about that.

Reply:
All right, although I will not be able to use fl_chart library without it.

Reply:
We can not add all features in a single PR, I said multiple times that I will add it after your pR, it should add in a separate phase, because I want to reuse it on other parts such as grid and borders, this is a public library and we should implement all features generic and customizable, (for example add LineStyle on other parts).
be patient